stream_alert_cli.py

```diff
@@ -123,6 +123,11 @@ def build_parser():
         help='Names of rules to test, separated by spaces'
     )
 
+    lambda_parser.add_argument(
+        '-l', '--live',
```

Comment:
DOES THIS STORE A BOOLEAN? MAYBE YOU SHOULD USE STORE_TRUE

(My caps was on what I started typing and I thought it'd be funny to just keep going)